src/main/java/org/openstreetmap/atlas/checks/base/BaseCheck.java

```diff
@@ -340,6 +340,12 @@ protected CheckFlag createFlag(final Set<AtlasObject> objects, final String inst
                 Collections.singletonList(instruction), points);
     }
 
+    protected CheckFlag createFlag(final Set<AtlasObject> objects, final List<String> instructions,
```

Comment:
minor ask, can you also implement an overloaded version without the need for locations which mirrors the other CheckFlag constructor.

Reply:
Did you one better and added that, plus the equivalent solo AtlasObject methods!

Reply:
huge